desc/plotting.py

```diff
@@ -1511,6 +1647,10 @@ def plot_boundaries(eqs, labels=None, zeta=None, ax=None, **kwargs):
         Z = coords["Z"].reshape(
             (grid.num_theta, grid.num_rho, grid.num_zeta), order="F"
         )
+
+        plot_data[f"R_{i}"] = R
```

Comment:
would it make more sense to return a dict of lists that matches up with the list of inputs?